src/main/scala/viper/gobra/frontend/info/implementation/typing/MemberTyping.scala

```diff
@@ -27,4 +27,14 @@ trait MemberTyping extends BaseTyping { this: TypeInfoImpl =>
       idenNumMsgs ++ constExprMsgs
     case s: PActualStatement => wellDefStmt(s).out
   }
+
+  private[typing] def wellDefBothMember(member: PMember): Messages = member match {
+    case m: PWithParam => wellDefVariadicArgs(m.args)
+    case _ => noMessages
+  }
```

Comment:
This seems a bit confusing to me. How do I determine whether I add a check here or in wellDefActualMember? I would just add the additional check to FunctionDecls and MethodDecls.

Reply:
My idea was to have a single method that is applied to both kinds of Members (actual and ghost) in the MemberTyping.scala file. The alternative would be to have these checks in `wellDefActualMember` and in `wellDefGhostMember`, which is the way I had before and which you advised against (because these methods are defined in `GhostMemberTyping`)